### PR TITLE
Promote rRSE job advert

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _posts/2022-05-12-rrse.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+3. **Make a difference.** Our team works with public /private sector organisations, and charities to give our output the best chance of making a positive difference outside the university. We also enthuse about and participate in open research wherever possible.
+4. **Flexibility.** We're happy to discuss flexible and part time working options. One of our team has recently reduced their hours from 100% to 80%. Working as a pool makes this much more practicable.Furthermore, in contrast to most RA roles this position is open-ended.
+5. **Do useful stuff with code.** We are most interested in your ability to do useful things with code and work with other people. We don’t tend to ask questions about the theory of computer science in interviews. The interest is in modern software technology, code you’ve written, how it was used and how it could be improved.
+6. **Reward and recognition** All of the Sheffield RSE team are employed within the academic careers pathway, a new flexible career development structure which recognises a broad range of research outputs including software. The pathway provides a clear route for promotion up to the level of professor.
```
Comment (davidwyld) [inline _posts/2022-05-12-rrse.md]:
```suggestion
6. **Reward and recognition.** All of the Sheffield RSE team are employed within the academic careers pathway, a new flexible career development structure which recognises a broad range of research outputs including software. The pathway provides a clear route for promotion up to the level of professor. 
```

Period at the end of bold text, for consistency.

File: _posts/2022-05-12-rrse.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+1. **Varied projects.** We work on research in areas such as astrophysics, healthcare, molecular biology, civil engineering and many more. Maybe you've been working on one area for years and you're keen to work with researchers in diverse fields?
+2. **We don't need an expert in everything right away.** Lots of people have joined our team having developed coding skills working on research, and learned more software engineering once in the team. Our team is made up of people who specialise in sharing software engineering skills with researchers.
+3. **Make a difference.** Our team works with public /private sector organisations, and charities to give our output the best chance of making a positive difference outside the university. We also enthuse about and participate in open research wherever possible.
```
Comment (davidwyld) [inline _posts/2022-05-12-rrse.md]:
```suggestion
3. **Make a difference.** Our team works with public/private sector organisations, and charities to give our output the best chance of making a positive difference outside the university. We also enthuse about and participate in open research wherever possible.
```

**Review by davidwyld — CHANGES_REQUESTED** [review]
Couple of typographic suggestions.


Do we need a call to action at the end? For instance "apply here" with another link to the advert.

Can we give an email address for questions as well?

**Comment by bobturneruk** [comment]
Good spots and good points. Thanks!

**Pull request merged**